distribution/changelog.txt

```diff
@@ -5,6 +5,7 @@
 - Feature: [#7956, #7964] Add sprite font glyphs for Hungarian and some Czech letters.
 - Feature: [#7971] Toolbox option to open custom content folder.
 - Feature: [#7980] Allow data path for RCT1 to be specified by a command line argument.
+- Feature: New console variable "current_rotation" to get or set view rotation
```

Comment:
Add the PR number and a full stop.

`- Feature: [#8080] New console variable "current_rotation" to get and set view rotation.`